packages/genesys-spark-components/src/components/stable/gux-time-picker/gux-time-picker.service.ts

```diff
@@ -143,7 +143,10 @@ export function getDisplayValue(
   const [hour, minute] = value.split(':');
 
   if (clockType === '12h') {
-    return `${(parseInt(hour, 10) % 12 || 12).toString()}:${minute}`;
+    const parsedHour = parseInt(hour, 10) % 12 || 12;
+    const leadingZero = parsedHour < 10 ? '0' : '';
+    const fullHour = `${leadingZero}${parsedHour.toString()}`;
+    return `${fullHour}:${minute}`;
```

Comment:
you can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart
```
const parsedHour = parseInt(hour, 10) % 12 || 12;
const fullHour = parsedHour.toString().padStart(2, '0');
```

Reply:
done